spec/requests/providers/offline_accounts_spec.rb

```diff
@@ -38,21 +38,12 @@
           end
         end
 
-        context "when applicant owns home with shared ownership" do
-          before { get providers_legal_aid_application_means_percentage_home_path(application) }
+        context "when applicant owns home" do
+          before { get providers_legal_aid_application_means_property_details_path(application) }
 
           it "points to percentage owned page" do
```

Comment:
One *tiny* nitpick... the test description should change to property details

Reply:
Ah yes good spot, thank you :)